### PR TITLE
Replace Google Custom Search with Swiftype

### PR DESCRIPTION
**Pull request by magwalk**
Since Google is EOL-ing it's Google Custom Search, we are replacing all of our site searches on our marketing sites with Swiftype. This adds the required js snippets and markup, as well as some CSS overrides. 
screen shot 2017-07-05 at 1 09 54 pm

### CODE REVIEW

**Review by hannah-radish — APPROVED** [review]
This is beautiful! ❤️  Lovely, thank you so much!

**Review by adamleff — APPROVED** [review]
Approved pending DCO sign-off and Travis going green again. Thank you @magwalk!

**Comment by adamleff** [comment]
All tests green! Thank you again, @magwalk! This will go live sometime Thursday afternoon ET.

**Pull request merged**
